README.md

```diff
@@ -2,145 +2,69 @@
 
 Operator for the infra.watch [smart gateway](https://github.com/redhat-service-assurance/smart-gateway)
 
+NOTE: Development environment for OCP 4.2 controlled from Service Assurance Operato (SAO)
+
+## Required CSV fields
+
+More information about ClusterServiceVersion fields required for Operator Lifecycle Manager that
+are not identified in the CSV Custom Resource Definition can be found here:
+
+> https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md
+
 ## Deployment to an existing cluster
 
-Deploy the components under the `deploy/` directory.
+Use the OperatorHub deployment from the Community catalog.
 
-```shell
-oc login -u system:admin
-oc apply -f deploy/operator.yaml
-oc apply -f deploy/crds/smartgateway_v1alpha1_smartgateway_crd.yaml
-oc apply -f deploy/role_binding.yaml
-oc apply -f deploy/role.yaml
-oc apply -f deploy/service_account.yaml
-```
+For development or latest version, create an OperatorSource for quay.io/redhat-service-assurance/smartgateway-operator.
 
-## Build and test against minishift
+## Build and test against CodeReady Containers
 
-A procedure for testing in minishift
+NOTE: You'll need a Red Hat subscription to access CodeReady Containers. Access of the pull secret
+is available at https://cloud.redhat.com/openshift/install/crc/installer-provisioned. Upstream CodeReady
+development is available at https://github.com/code-ready/crc.
 
-Tested with the following versions:
-* docker
-  * 18.06.0-ce
-  * 18.09.7
+A procedure for testing in CodeReady Containers will be available soon.
+
+* buildah
+  * <todo>
```

Comment:
I kinda object to a patch that goes from "This works as tested on X.Y.Z" to "I dunno what version this works on - does it?"